plugins/kubernetes/app/models/kubernetes/resource_template.rb

```diff
@@ -177,6 +177,11 @@ def set_env
         env << {name: k, value: template[:spec][:template][:metadata][:labels][k.downcase].to_s}
       end
 
+      # name of the cluster
+      [:CLUSTER_NAME].each do |k|
```

Comment:
why the each ?
there are lots of clusters ... call it KUBERNETES_CLUSTER_NAME ?